05_syntax_lab/01.py

```diff
@@ -1,5 +1 @@
-"""
-Write a program that reads 10 numbers from
-the user and prints the largest one
-"""
-
+solution for lab 1
```

Comment:
Very good solution.
Next time be sure to add comments and you can improve by doing proper error handling